com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -1502,6 +1521,7 @@ static class ShaderConstants
             public static readonly int _Chroma_Params = Shader.PropertyToID("_Chroma_Params");
             public static readonly int _Vignette_Params1 = Shader.PropertyToID("_Vignette_Params1");
             public static readonly int _Vignette_Params2 = Shader.PropertyToID("_Vignette_Params2");
+            public static readonly int _Vignette_Params3 = Shader.PropertyToID("_Vignette_Params3");
```

Comment:
Maybe call this "_Vignette_ParamsXR" since that's what it is and it's easier to read later.

Reply:
Yes, we can do that